static/ca_bydate.js

```diff
@@ -110,6 +110,9 @@ function seriesToPlot() {
     results = _.filter(results, function(series) {
       return series.data.length > 0;
     });
+
+    results.unshift({ name: 'Show/Hide All', visible: false });
+
```

Comment:
this is really evil and clever, @robertobuso

Reply:
:)